dist/isoCurrency.js

```diff
@@ -894,8 +894,16 @@ angular.module('isoCurrency', ['isoCurrency.common'])
 			if (!currency) {
 				return amount;			
 			}
+
 			var fractionSize = (fraction === void 0) ? currency.fraction : fraction;
-			return $filter('currency')(amount, currency.symbol, fractionSize);
+	        var currencyFilter = $filter('currency');         
+
+	        if(amount < 0){
```

Comment:
Comment would be good here.  Particularly if there's a design doc or someone that said "it should be this way", definitely write it down.  This is the kind of shit that someone says later" this is wrong" and we need some accountability of why it is this way